Testscripts/Linux/containers_utils.sh

```diff
@@ -69,7 +69,7 @@ function VerifyDockerEngine() {
 # Function to start docker engine
 function StartDockerEngine() {
     LogMsg "Start docker engine"
-    systemctl start docker || service docker start
+    systemctl restart docker || service docker restart
```

Comment:
Can we check if it's started, if it's not, then start it? I'm not sure if there is problem on restart.

Reply:
Johnson confirmed that changing from start->restart does not hurt here.

Reply:
ok

Reply:
I still think @squirrelsc comment is higher value